linera-rpc/src/simple/client.rs

```diff
@@ -140,6 +141,14 @@ impl ValidatorNode for SimpleClient {
         self.query(RpcMessage::DownloadBlob(Box::new(blob_id)))
             .await
     }
+
+    async fn download_certificate(
+        &mut self,
+        hash: CryptoHash,
+    ) -> Result<CertificateValue, NodeError> {
```

Comment:
Should this be a `Certificate` (including signatures), rather than just the value?

Reply:
Discussed on the coworking meeting: We probably need both, so there's a following PR with `Certificate` also